mscore/menus.cpp

```diff
@@ -83,6 +83,7 @@
 #include "libmscore/vibrato.h"
 #include "libmscore/palmmute.h"
 #include "libmscore/fermata.h"
+#include <libmscore/measurenumber.h>
```

Comment:
Why not use `#include "libmscore/measurenumber.h"` just like everything above?

Reply:
Agreed

Reply:
agreed

Reply:
haha, I really find it funny how dumb I can be sometimes :)